code/controllers/subsystem/supply_shuttle.dm

```diff
@@ -482,15 +482,13 @@ var/datum/subsystem/supply_shuttle/SSsupply_shuttle
 
 /datum/subsystem/supply_shuttle/proc/forbidden_atoms_check(atom/A)
 	var/contents = get_contents_in_object(A)
-	for(var/mob/living/simple_animal/hostile/mimic/M in contents)
-		M.angry = 0
-		M.apply_disguise()
 	for(var/mob/living/M in contents)
-		if(M.key || M.ckey) //only mobs that were never player controlled
+		if(istype(M, /mob/living/simple_animal/hostile/mimic))
+			var/mob/living/simple_animal/hostile/mimic/mimic = M
+			mimic.angry = 0
+			mimic.apply_disguise()
+		if(M.key || M.ckey || M.mind) //only mobs that were never player controlled
 			return TRUE
-		for(var/datum/mind/M2 in ticker.minds) //see above, but for ghosts
```

Comment:
Why remove this? I don't get why you feel the need to touch this file, anyway, it's not like it contains any redundant `ticker.minds += xyz...` which is what we were discussing.